src/main/java/nextstep/courses/strategy/PaidPaymentStrategy.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.courses.strategy;
+
+import nextstep.payments.domain.Payment;
+
+public class PaidPaymentStrategy implements PaymentStrategy {
+    private final long price;
+
+    public PaidPaymentStrategy(long price) {
+        this.price = price;
+    }
+
+    @Override
+    public boolean payable(Payment payment) {
+        if (payment.isSameAmount(price))
+            return true;
+        return false;
```

Comment:
```suggestion
         return payment.isSameAmount(price);
```

쉽게 표현할수있는것은 쉽게 표현해도 좋을것 같아요~